bin/jenkins/functions.sh

```diff
@@ -124,33 +124,45 @@ selenium_tests () {
   return $rc
 }
 
+deploy () {
+  PROJECT=$1
+  print_line
+  echo "Running Deploy"
+  docker exec -i ${PROJECT}_dev_1 su vagrant -c "
+    source /vagrant/bin/init_vagrant_env &&
+    make clean_appengine > /dev/null &&
```

Comment:
Some commands like `deploy_appengine deploy_settings_grc_dev.sh` output sensitive information (API keys and such) and because the log is publicly available on our jenkins machine I don't want this to be printed out at all.

I'm only redirecting standard output so errors will still show up unless I do something like `2>`, which I'm also considering because you never know when an error will leak something you don't want the world to see :)